frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.styled.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from "@emotion/styled";
+import type { ReactNode } from "react";
+import { color } from "metabase/lib/colors";
+import { Box, Center, Text } from "metabase/ui";
+import { Icon } from "metabase/core/components/Icon";
+
+const ContentWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: stretch;
+
+  min-height: 648px;
+`;
+
+const SettingsAsideBlock = styled.div`
+  flex-shrink: 0;
+  width: 345px;
```

Comment:
Do we have better proxies for width here besides this really specific pixel value (similar to line 12 and 25 for this file too). At the very least, let's `rem` values, but maybe we can think about how wide we want that sidebar to be on different screen sizes and be a little smarter about it - maybe something like 30vw or a `min/max`. I wouldn't think about it _too_ much though, it's a low-value add